pyreisejl/utility/call.py

```diff
@@ -38,6 +91,9 @@ def main(args):
         args.input_dir = scenario_args[3]
         args.execute_dir = scenario_args[4]
 
+        pkl_to_csv(args.input_dir)
+        pkl_to_case_mat(args.input_dir)
```

Comment:
Is this included for `save_input_mat`?

Reply:
Yep - I think that's the only thing that still uses case.mat.

Reply:
Great, thanks! I believe we discussed it a week ago, but can you remind me why we still need to generate a `case.mat` file at all?

Reply:
Sounds good! Thank you for clarifying!